CRM/Activity/BAO/Activity.php

```diff
@@ -1017,7 +1017,7 @@ public static function createEmailActivity($sourceContactID, $subject, $html, $t
    * @param int $caseId
    *
    * @return array
-   *   ( sent, activityId) if any email is sent and activityId
+   *   ( sent, activityIds) if any email is sent and the corresponding activity ids
```

Comment:
Good point.

Reply:
I'm going to look at the $sent issue separately since it's been like that since [2009](https://github.com/civicrm/civicrm-svn/commit/493e9d1d52e13cb2a60dc8fb3db872b79c2226c3#diff-205800b0086e4e7f73b39a41f3aa9760637718d32ab4fc9424efc3bb0f8606faL1002). Kudos for noticing.